### PR TITLE
docs: add a single page with all API setup instructions

### PR DESCRIPTION
**Pull request by andrew-stark-cko**
## Description

The current docs have code to set up processout.js in several places but with no obvious "home" that a user can bookmark. The iOS/Android client page includes some of the SDK installation advice only in code comments and the server API setup (only) is kept separately in the API reference docs. Also, the docs frequently show public/private access keys (production and sandbox versions) when instantiating the ProcesssOut object but there is no explanation anywhere about how you obtain these keys.

It would be useful if all of the API installation, setup and access information was collected together in one page along with instructions for generating and using the access keys. The inline references in pages could then be linked to this page.

## Solution

Add the API access page with all the relevant information for all platforms. Update the testing/sandbox page to link explicitly to the newly-added sandbox setup instructions. Replace existing inline text/code about setup with links to appropriate sections of the API access page.

## Notes

I also replaced "test environment" with "sandbox" throughout the docs and fixed a couple of presentation issues (adding non-breaking spaces to links and Dashboard UI "paths".

In a couple of places, the JS pages mention adding the example in a separate script tag to the one where processout.js is loaded. Do we actually need to remind users that this is necessary? Have users had problems because of this in the past, maybe?

### CODE REVIEW

File: source/payments/googlepay.md.erb [hunk 1]
```diff
@@ -116,14 +116,10 @@ client.tokenize(card, new TokenCallback() {
...
 ### Web
-We provide [ProcessOut.js](https://docs.processout.com/payments/processoutjs-reference/)
-which can handle creating a Google Pay™ token for you. In order to make use of ProcessOut.js
-you must first load the javascript on your page. It is mandatory for regulation reasons to always load it from our
-CDN as shown below, to ensure you always have the latest version.
 
-```html
-<script src="https://js.processout.com/processout.js"></script>
-```
+Our `processout.js` library can create a Google Pay™ token for you.
```
Comment (louis-paul-dareau-cko) [inline source/payments/googlepay.md.erb:120]: (minor) Not sure we need the TM for Google Pay.

File: source/payments/refresh-card-cvc.md.erb [hunk 2]
```diff
@@ -11,14 +11,18 @@ able to make use of it.  While this guarantees the security of the customer's
...
 the main difference being you'll re-use an already existing Card token to be refreshed.
 
-Let's first set up the form that'll be used to collect the CVC code. Similarly
+Firstly, follow the [API access instructions](/api-access/) to
+enable the `processout.js` library in your web page and set up the
+access keys for your project. We recommend you use the
+[sandbox](/api-access/#enabling-the-sandbox) for safety during
+experimentation and testing.
+
+Let's set up the form that'll be used to collect the CVC code. Similarly
```
Comment (louis-paul-dareau-cko) [inline source/payments/refresh-card-cvc.md.erb:25]: “CVC code” is redundant as CVC means Card Verification Code.

File: source/api-access.md.erb [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+
+Start by loading the `processout.js` library in your web
+page. 
+
+```html
+<script src="https://js.processout.com/processout.js"></script>
+```
+
+**Note:** for security purposes, it is *very* important
+that you access this file only from our content delivery network
```
Comment (louis-paul-dareau-cko) [inline source/api-access.md.erb:80]: (minor)

```suggestion
that you serve this file only from our content delivery network
```

File: source/payments/refresh-card-cvc.md.erb (hunk 2 above)
Comment (andrew-stark-cko) [reply source/payments/refresh-card-cvc.md.erb]: All instances now removed.

File: source/api-access.md.erb (hunk 3 above)
Comment (andrew-stark-cko) [reply source/api-access.md.erb]: Fixed.

**Review by louis-paul-dareau-cko — APPROVED** [review]

**Pull request merged**